src/darwin/Framework/CHIP/MTRDeviceControllerFactory.mm

```diff
@@ -1012,9 +932,6 @@ - (void)resetOperationalAdvertising
         });
     }
 }
-@end
-
-@implementation MTRDeviceControllerFactory (InternalMethods)
```

Comment:
I think we were trying to indicate which things were not really part of the API... But maybe we should just move to using `_` prefixes for that?  Anyway, that's all followup bits.

Reply:
Hm yeah I'm not sure trying to keep all the internal methods in an actual category implementation is worth the hassle. Similarly, the underscore convention can become tedious if something changes from being internal to public means having to rename it. I think in practice it's often a case of just having to look which header (if any) a method is declared in (hopefully the IDE provides some quick way of jumping between implementation and declaration of a method -- Cmd-Click in Xcode).